spec/controllers/application_controller_spec.rb

```diff
@@ -37,20 +37,28 @@ def invalid_authenticity_token
     end
   end
 
-  it "does not force ssl if LOCAL_HTTPS is not 'true'" do
-    routes.draw { get 'success' => 'anonymous#success' }
-    ENV['LOCAL_HTTPS'] = ''
-    allow(Rails.env).to receive(:production?).and_return(true)
-    get 'success'
-    expect(response).to have_http_status(:success)
-  end
+  context do
+    around do |example|
+      original_local_https = ENV['LOCAL_HTTPS']
```

Comment:
It looks like we have the climate control gem in the repo - https://github.com/thoughtbot/climate_control - but we are not using it in many places. Do you want to change this `around` to use climate control instead of the explicit before/after assignment here?